pkg/ui/workspaces/cluster-ui/src/insightsTable/insightsTable.tsx

```diff
@@ -93,7 +93,11 @@ export const insightsTableTitles: InsightsTableTitleType = {
 };
 
 function typeCell(value: string): React.ReactElement {
-  return <div className={cx("insight-type")}>{value}</div>;
+  let insightType = "insight-type";
+  if (value === "Failed Execution") {
+    insightType = "insight-type-failed";
+  }
+  return <div className={cx(insightType)}>{value}</div>;
```

Comment:
nit, replace wtih:
```
const className = value === "FailedExecution" ? "insight-type-failed" :  "insight-type";
```
Can you do me a favour and refactor this function 🙏 :
- capitalize this function also, since it's returning a component
- Change the param to take the InsightTypes enum instead of just a string, do the enum to string conversion in this function instead of the call site

Reply:
Done. The function looks way cleaner now!